recipe/meta.yaml

```diff
@@ -39,10 +41,6 @@ requirements:
 test:
   imports:
     - lsst_efd_client
-  commands:
-    - pip check
```

Comment:
And this is because lsst-efd-client pins numpy to something that would be massively over-constrained in a conda-forge recipe, correct?  If so, it's also regrettable and something we should remember to put back in later.  In fact, it may be worth commenting this out with a notation that it can come back when upstream is less constrained, instead of removing it.

Reply:
Basically, I think that was the problem. It's also a case that pip and conda-forge check and set up dependencies differently, and what was happening here was a little odd.